_maps_2024/01-delmelle.md

```diff
@@ -0,0 +1,9 @@
+---
+day: 1
+creator: Elizabeth Delmelle
+title: Oh the Places I've Lived
+thumbnail: 
+source: https://github.com/ECDelmelle/30DayMapping24/blob/main/Scripts/Day1_Points.R # Add a link to your source code here
+---
+
+By default, your thumnail will be displayed here. If that's good enough, cool, just delete all of this text below the front matter.
```

Comment:
Hey @ECDelmelle, would you do me a favor and just get rid of these lines? That way your image will show up on the page. 


```suggestion
```